frontend/src/routes/profile/+page.svelte

```diff
@@ -78,7 +78,7 @@
 			monthChart.update();
 		}
 	}
-	let year = [2020, 2021, 2022, 2023];
+	let year = [2020, 2021, 2022, 2023, 2024];
```

Comment:
better support more commonly used year
```suggestion
	let year = [2020, 2021, 2022, 2023, 2024, 2077];
```